config.yml

```diff
@@ -16,9 +16,10 @@
 ########################################
 
 ########################################
-# OpenSUSE Tumbleweed
+# openSUSE Tumbleweed
+# vagrant box add --provider libvirt --name virt-appl/Tumbleweed https://download.opensuse.org/repositories/Virtualization:/Appliances:/Images:/openSUSE-Tumbleweed/openSUSE_Tumbleweed/Tumbleweed.x86_64-libvirt.box
 ########################################
-opensuse/openSUSE-Tumbleweed-x86_64:
+virt-appl/Tumbleweed:
```

Comment:
Why not keeping the `opensuse/` prefix?

Reply:
Well, I thought that I should leave some sort of hint on the name of the Virtualization Appliances repo, where the box is coming from.

Reply:
I think we could leave standard Tumbleweed box from cloud, it is the same as in virt.